project/src/components/app/app.jsx

```diff
@@ -11,12 +11,17 @@ import offerProps from '../offers/offer.props';
 
 function App(props) {
   const { offers } = props;
+  const [currentPoint, setCurrentPoint] = useState({});
```

Comment:
```suggestion
  const [currentPoint, setCurrentPoint] = useState(null);
```
Давай либо null, либо нормальный оффер